src/backend/distributed/operations/shard_cleaner.c

```diff
@@ -245,20 +615,18 @@ TryLockRelationAndPlacementCleanup(Oid relationId, LOCKMODE lockmode)
  * true on success.
  */
 static bool
-TryDropShard(GroupShardPlacement *placement)
+TryDropShardOutsideTransaction(char *qualifiedTableName, char *nodeName, int nodePort)
 {
-	ShardPlacement *shardPlacement = LoadShardPlacement(placement->shardId,
-														placement->placementId);
-	ShardInterval *shardInterval = LoadShardInterval(shardPlacement->shardId);
-
-	ereport(LOG, (errmsg("dropping shard placement " INT64_FORMAT " of shard "
-						 INT64_FORMAT " on %s:%d after it was moved away",
-						 shardPlacement->placementId, shardPlacement->shardId,
-						 shardPlacement->nodeName, shardPlacement->nodePort)));
+	char *operation = (CurrentOperationId == INVALID_OPERATION_ID) ? "move" : "cleanup";
```

Comment:
should probably get the operation ID as a parameter, for the maintenance daemon

Reply:
Yes, thanks for pointing out. Will fix.